src/shared/ui/form/form-error-message.tsx

```diff
@@ -0,0 +1,10 @@
+import { cn } from '../../utils'
+
+interface Props {
+  message: string
+  className?: string
+}
+
+export default function FormErrorMessage({ message, className }: Props) {
```

Comment:
```suggestion
export default function FormErrorMessage({ children, className, ...restProps }: HTMLAttributes<HTMLParagraphElement>) {
```

이렇게 사용하면 `message` prop 대신에 children으로 보여줄 수 있을것 같아요.

그리고 `className`을 일일이 작성하는것보다 타입을 확장해서 사용하는게 더 좋아보입니다.

Reply:
저도 message 작성해놓고 children이 낫나 잠깐 고민했었는데 이게 더 낫겠군요 ㅎㅎ